gulpfile.js

```diff
@@ -83,6 +93,13 @@ var webpackConfig = {
       }
     ]
   },
+  plugins: [
+    new webpack.NormalModuleReplacementPlugin(/\/config\/config(\.js)?$/,
+      function (result) {
+        result.request = configFilePath;
+      }
+    )
+  ],
```

Comment:
There are a number of unexpected (but welcome) changes to the gulpfile in this PR. Still, I would have preferred them in a separate one.